### ISSUE TITLE
SDR play support soon ?

### ISSUE THREAD

**Issue by m1ctk** [KejPi/AbracaDABra#158]
All working nicely but wonder when sdr play devices will be added ?

**Comment by KejPi** [comment]
Well, I do not have any SDR Play device. I would be difficult to add support them "blindly" without having the HW  :-(

**Comment by m1ctk** [comment]
Yes sure but there are some second hand ones on ebay I see or maybe you can borrow one etc. There are very very popular now amoung sdr users

**Comment by KejPi** [comment]
If someone offers to lend me his device for the time necessary to implement support for that in the application I will do it.

**Comment by m1ctk** [comment]
> If someone offers to lend me his device for the time necessary to implement support for that in the application I will do it.

But can you not afford to buy one at least second hand  ?!  Well worth it. I doub anyone would lend you one imo

**Comment by KejPi** [comment]
Is it not about whather I can afford it or not. I do not want to invest mora than 100 EUR to purchase a HW just to be able to add support for it in application that I give for free. And on top of that I doubt it is worth the money for static DAB reception, there is only marginal benefit compared to cheap RTL-SDR USB stick.

**Comment by m1ctk** [comment]
> Is it not about whather I can afford it or not. I do not want to invest mora than 100 EUR to purchase a HW just to be able to add support for it in application that I give for free. And on top of that I doubt it is worth the money for static DAB reception, there is only marginal benefit compared to cheap RTL-SDR USB stick.

Well that's shame as the rsp1a and the rsp dx are far more sensitive and selective than a RTL SDR in my experience when used with   Qirx. Hoping someone can lend you one for you to add support

**Comment by KejPi** [comment]
Well, it should be significantly better considering the price.
Have you tried with SoapySDR? AbracaDABra supports SoapySDR and SDR Play is supported https://github.com/pothosware/SoapySDRPlay3/wiki

**Comment by m1ctk** [comment]
> Well, it should be significantly better considering the price. Have you tried with SoapySDR? AbracaDABra supports SoapySDR and SDR Play is supported https://github.com/pothosware/SoapySDRPlay3/wiki

All to complex to set up. I hope you can simply add sdr play support eventually
